src/rulesets/oas/index.ts

```diff
@@ -1,7 +1,10 @@
 import { DiagnosticSeverity } from '@stoplight/types';
 import { FunctionCollection, RuleCollection, RuleFunction, RuleType } from '../../types';
 
-export const operationPath = "$..paths.*[?( name() !== 'parameters' )]";
+export const operationPath =
+  "$..paths.*[?( name() === 'get' || name === 'put' || name === 'post'" +
```

Comment:
```suggestion
  "$..paths.*[?( name() === 'get' || name() === 'put' || name() === 'post'
```
+ extra quote
+ I believe we need to call the `name` function